src/SmartComponents/SystemTable/SystemTable.js

```diff
@@ -51,6 +59,10 @@ const SystemTable = ({
     });
   }, [selectedIds]);
 
+  useEffect(() => {
```

Comment:
If what we want from this hook is to refresh the inventory table when eligibility changes, we can achieve it by passing the eligibility value to customFilters prop as [here](https://github.com/RedHatInsights/patchman-ui/blob/ab9b4d5aa4444da2caf816ef0cf679d694a12871/src/SmartComponents/Systems/SystemsTable.js#L144).

Reply:
Yeah I had a look at using customFilters but I couldn't figure out how it worked.  There was no documentation and all the examples I looked at didn't make sense to me, that's why I ended up doing it the way I did, because it made sense :)  I can have another look at it.

Reply:
customFilters confuse the heck out of me.  I'll leave the filters as they are for now and start playing around with customFilters.  I'll look to change them to customFilters when I get more knowledge/experience.